DB/src/main/java/io/deephaven/db/tables/Table.java

```diff
@@ -2287,7 +2287,7 @@ default Table sort(Collection<SortColumn> columnsToSortBy) {
     Table snapshot(Table rightTable, boolean doInitialSnapshot, String... stampColumns);
 
     default Table snapshot(Table rightTable, String... stampColumns) {
```

Comment:
Can you clarify the JavaDoc on `TableOperations` to specify the default behavior? Also, please make sure @devinrsmith and the dev-rel folks know about this change, for QST and documentation respectively.